gui/src/components/QuickFastModeToggle.tsx

```diff
@@ -15,6 +15,7 @@ export function QuickFastModeToggle() {
     <div className="flex items-center space-x-2">
       <Switch
         defaultChecked={fastMode}
+        checked={fastMode}
```

Comment:
same as above here. `defaultXXX` is different from `XXX`, in the sense that it won't update if the value changes (on purpose). this seems suspicious